bin/setup

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash -e
-docker-compose run start_services
-docker-compose run start_applications
+docker-compose --file ./tests/call-centre/docker-compose.yaml run start_services
+docker-compose --file ./tests/call-centre/docker-compose.yaml run start_applications
```

Comment:
What do you think about keeping the `docker-compose.yaml` in the root? This would be to keep describing the runnable services and dependencies in a single file until we are sure we need separate ones.